portal/config/eproms/Questionnaire.json

```diff
@@ -7843,6 +7843,23 @@
           "valueBoolean": true
         }
       ]
+    },
+    {
+      "identifier": [
+        {
+          "system": "http://us.truenth.org/questionnaire",
+          "value": "ironman_post_intervention"
```

Comment:
You might abbreviate intervention, perhaps "tx"?

Reply:
Good idea.  Now using `ironman_ss_post_tx`.